Orange/widgets/data/owtransform.py

```diff
@@ -31,6 +35,7 @@ class Error(OWWidget.Error):
     def __init__(self):
         super().__init__()
         self.data = None  # type: Optional[Table]
+        self.template_data = None  # type: Optional[Table]
```

Comment:
Since `self.template_data`  has been introduces `self.template_domain` is no longer needed.